filament/src/details/Engine.cpp

```diff
@@ -188,6 +217,13 @@ FEngine::FEngine(Backend backend, Platform* platform, void* sharedGLContext) :
     // (it may not be the case)
     mJobSystem.adopt();
 
+    const size_t MB = 1024U * 1024U;
```

Comment:
constexpr and call it `MiB` which is more accurate (MB refers to 1000x1000)

Reply:
Fixed in my next update.